product_search/main.py

```diff
@@ -0,0 +1,24 @@
+"""Return the inventory details for an item in the product inventory"""
+
+import sys
+from product_search import (
+    get_product_name,
+    item_inventory_details,
+    display_product_information,
+)
+
+
+def main(filename):
```

Comment:
@paulghaddad Is this the best way to set up the main function inside this main.py file?

Reply:
👍 see comment below